### ISSUE TITLE
Missing example of binary field usage

### ISSUE THREAD

**Issue by kokujin** [balderdashy/waterline-docs#22]
Is there some example somewhere? I ways thinking of storing thumbnails in my database using the binary field. Thanks

**Comment by devinivy** [comment]
Good point!  Each adapter may deal with this entirely differently.  I'll mark this to make that clearer.  Perhaps the docs could link to an example from an adapter.

If you learned anything about how to do this, or found any good examples, please let us know.
